js/filter-selection.js

```diff
@@ -0,0 +1,109 @@
+const MIN_VALUE = 0;
+const MAX_VALUE = 100;
+
+const radioButtons = document.querySelectorAll('.effects__radio');
+const preview = document.querySelector('.img-upload__preview');
+const sliderElement = document.querySelector('.effect-level__slider');
+const sliderValue = document.querySelector('.effect-level__value');
+const sliderContainer = document.querySelector('.effect-level');
+
+const filterOptions = {
+  'chrome': {
```

Comment:
ключи в объектах всегда строки поэтому кавычки избыточны
```suggestion
  chrome: {
```